src/Worker/Core/Shims/TaskOrchestrationContextWrapper.cs

```diff
@@ -38,9 +38,27 @@ public TaskOrchestrationContextWrapper(
         OrchestrationContext innerContext,
         OrchestrationInvocationContext invocationContext,
         object? deserializedInput)
+        : this(innerContext, invocationContext, deserializedInput, new Dictionary<string, object?>())
+    {
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="TaskOrchestrationContextWrapper"/> class.
+    /// </summary>
+    /// <param name="innerContext">The inner orchestration context.</param>
+    /// <param name="invocationContext">The invocation context.</param>
+    /// <param name="deserializedInput">The deserialized input.</param>
+    /// <param name="properties">The configuration for context.</param>
+    public TaskOrchestrationContextWrapper(
+        OrchestrationContext innerContext,
+        OrchestrationInvocationContext invocationContext,
+        object? deserializedInput,
+        Dictionary<string, object?> properties)
```

Comment:
Can this be read-only?
```suggestion
        IReadOnlyDictionary<string, object?> properties)
```